docs/Validation-and-Serialization.md

```diff
@@ -163,13 +163,19 @@ const bodyJsonSchema = {
 }
 
 const queryStringJsonSchema = {
-  name: { type: 'string' },
-  excitement: { type: 'integer' }
+  type: 'object',
```

Comment:
I would revert this change because this example was on purpose.
It is written before the code:

> or a simpler variation in which the `type` and `properties` attributes are forgone and the parameters are listed at the top level (see the example below).

Reply:
Then maybe have both examples? I can add the original back and also the current one?